.github/workflows/build.yml

```diff
@@ -2,22 +2,56 @@ name: Build
 
 on: [ pull_request ]
 
+env:
+  NODE_ACTIVE_LTS_VERSION: 24
+  NODE_PREVIOUS_LTS_VERSION: 22
+  NPM_VERSION: 11
+
 jobs:
+  prepare_env:
```

Comment:
`prepare_env` is need as `matrix`  does not allow to access `env` as `steps` does.